scipy/linalg/setup.py

```diff
@@ -146,8 +146,14 @@ def configuration(parent_package='', top_path=None):
     config.add_extension('_decomp_update',
                          sources=['_decomp_update.c'])
 
+    config.add_data_files('_cythonized_array_utils.pxd')
+    
     config.add_extension('_cythonized_array_utils',
-                         sources=['_cythonized_array_utils.c'])
+                         sources=['_cythonized_array_utils.c'],
+                         depends=['_cythonized_array_utils.pyx',
+                                  '_cythonized_array_utils.pxd'],
+                                  include_dirs=['.']
```

Comment:
I am not sure if this line is needed. I got varying results and I certainly don't understand anything that is going on here. But this started to give consistent results.

Reply:
Yes indeed, it's needed with `distutils`. For Meson it never is; the current source directory is automatically added.